.github/workflows/test_nginx_lastest_commit.yml

```diff
@@ -10,8 +10,13 @@ jobs:
   test-nginx:
     strategy:
        fail-fast: false
+       matrix:
+          compiler:
+            - { compiler: LLVM, CC: clang, CXX: clang++}
     runs-on: "ubuntu-20.04"
-
+    env:
+       CC: ${{ matrix.compiler.CC }}
+       CXX: ${{ matrix.compiler.CXX }}
```

Comment:
it worked.

![image](https://user-images.githubusercontent.com/623062/217757508-31e9acd3-1150-44c1-bbb7-9b55a64550b0.png)